cypress/support/pages/shippingMethodPage.js

```diff
@@ -189,16 +189,16 @@ export function saveRateAfterUpdate() {
 
 export function fillUpLimits({ max, min }) {
   cy.get(SHIPPING_RATE_DETAILS.minValueInput)
-    .type(min)
+    .type(min, { force: true })
```

Comment:
**question**: `{ force: true }` skips all the checks,  e.g. if element's invisible or disabled. Why do we use it so much here?

Reply:
Good point Droniu. PriceField uses a new component from Macaw, which when input is empty shows a label, that label hides input, I that cause problem because I can not click on input using cypress.